LondonBritishLibrary/orient/BLorient739.xml

```diff
@@ -132,17 +132,20 @@ schematypens="http://relaxng.org/ns/structure/1.0"
             <textClass>
                 <keywords>
                     <term key="Commentary"/>
-                    
+                    <term key="ChristianLiterature"/>
                 </keywords>
             </textClass>
-            <langUsage/>
+            <langUsage><language ident="gez">Gǝʿǝz</language></langUsage>
```

Comment:
and "en"?

Reply:
I had some time agp asked @PietroLiuzzo whether this was necessary and didn't receive an answer...I vaguely remember that it doesn't matter, but a confirmation would be nice, as the question comes up often!

Reply:
Sorry, I do not recall that question, I apologise if I did not reply promptly enough or missed it entirely, or caused unwillingly reasons to assume other offending behaviour. [`<language>`](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-language.html)s and their `@ident` in `<langUsage>` should resolve the reference for any `@xml:lang` in the TEI file. [When these where automatically updated to remove the extra values all values attested in the file where added, those absent removed.](https://github.com/BetaMasaheft/Documentation/issues/1243)  In the [Guidelines](https://betamasaheft.eu/Guidelines/?id=languageIntro) it is pointed out how the presence of extra values is not a problem, like in this case, where there is no `xml:lang="gez"` in the file. This is especially true where we have no text at all, because it will be that little bit of information allowing to get the text searching by language. You do have a `@xml:lang='en'` but that is the standard value for all our TEI files. So, to be correct following the above guidance, you should have en as well, but it is not used by the app for example.

Reply:
Thank you! I'll merge this as it is and make a new PR to add English